docs/source/user/api.rst

```diff
@@ -248,8 +248,13 @@ obtaining access tokens, validating their phone number, etc.).
   `Organization API Token <#organization-api-token>`_
   described in the beginning of this document.
 
-  Some of them require the sending of the user API access token
-  sent in the form of a "Bearer Token".
+Some endpoints require the sending of the user API access
+token sent in the form of a "Bearer Token", example:
+
+.. code-block:: shell
+
+    curl -H "Authorization: Bearer <user-token>" \
+         'http://localhost:8000/api/v1/default/account/session/'
```

Comment:
Removed example from below and added it before all User APIs as the example to use `Bearer <user-token>` is applicable for multiple User APIs